bin/compile

```diff
@@ -18,7 +35,17 @@ cd $BUILD_DIR
 if [ -f shard.yml ]; then
   echo "-----> Installing Dependencies"
   $CRYSTAL_DIR/bin/crystal deps
+
+  if [ -f app.cr ]; then
+    echo "-----> Compiling app.cr (overriden - app.cr exists)"
```

Comment:
I don't think we need the extra info here. If you already have a `shard.yml` and an `app.cr`, there is no more you have to do.